packages/core/src/css/cssConfig.ts

```diff
@@ -0,0 +1,145 @@
+import { createRequire } from 'node:module';
+import path from 'node:path';
+import type { RsbuildConfig, RsbuildPlugin } from '@rsbuild/core';
+import { CSS_EXTENSIONS_PATTERN } from '../constant';
+import { RemoveCssExtractAssetPlugin } from './RemoveCssExtractAssetPlugin';
+
+const require = createRequire(import.meta.url);
+
+export const RSLIB_TEMP_CSS_DIR = '__rslib_css__';
+
+// https://rsbuild.dev/zh/config/output/css-modules#cssmodulesauto
+export type CssLoaderOptionsAuto =
+  | boolean
+  | RegExp
+  | ((
+      resourcePath: string,
+      resourceQuery: string,
+      resourceFragment: string,
+    ) => boolean);
+
+export function isCssFile(filepath: string): boolean {
+  return CSS_EXTENSIONS_PATTERN.test(filepath);
+}
+
+const CSS_MODULE_REG = /\.module\.\w+$/i;
+export function isCssModulesFile(
+  filepath: string,
+  auto: CssLoaderOptionsAuto,
+): boolean {
+  const filename = path.basename(filepath);
+  if (auto === true) {
+    return CSS_MODULE_REG.test(filename);
+  }
+
+  if (auto instanceof RegExp) {
+    return auto.test(filepath);
+  }
+
+  if (typeof auto === 'function') {
+    return auto(filepath, '', '');
```

Comment:
no resourcePath and resourceQuery?

Reply:
done, by copy the parsing function from Rspack